### PR TITLE
Upload page

### LINKED ISSUE #15: Upload Media functionality
Issue #12 

In our form, on the 4th page, we would like the user to have the option to upload a media (picture, video or recording) so we should start thinking on how we will implement this.

### PR DESCRIPTION
**Pull request by MynahMarie**
Closes #15 

In this PR we have implemented the upload image functionality using a third party.

### CODE REVIEW

**Review by MarlenAw — APPROVED** [review]

**Review by hoslack — APPROVED** [review]

File: client/src/config/keys.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
+CURRENCY_API_KEY: '231eb90015c1884b34ef94c0c0c3b4bc',
+IMAGE_UPLOAD_PRESET: 'i50l2x2r',
+IMAGE_API_SECRET: 'S253wP6tc6hhr2G69DzzWdpi7Gw',
```
Comment (Karyum) [inline client/src/config/keys.js]: ummm ? i can see the API keys

Comment (Karyum) [reply client/src/config/keys.js]: there is a way in create react app to make into an env variable [check this](https://medium.com/@tacomanator/environments-with-create-react-app-7b645312c09d)

File: client/src/containers/UploadForm.js [hunk 2]
```diff
@@ -32,6 +38,44 @@ class UploadForm extends Component {
...
+
+    uploadRequest.end((err, res) => {
+      if (err) {
+        alert(err);
+        return;
+      }
+
+      this.setState({ imageUrl: res.body.url });
+      console.log('Upload Successful', JSON.stringify(res.body.url));
+      console.log('State:', this.state);
```
Comment (Karyum) [inline client/src/containers/UploadForm.js]: logs in the frontend are not great :sweat_smile:

Comment (MynahMarie) [reply client/src/containers/UploadForm.js]: @Karyum took out console logs and cleaned up the config files!

**Review by Karyum — APPROVED** [review]
# :heart:

**Pull request merged**